docs/_includes/js/carousel.html

```diff
@@ -4,20 +4,20 @@ <h1 id="carousel" class="page-header">Carousel <small>carousel.js</small></h1>
   <h2 id="carousel-examples">Examples</h2>
   <p>The slideshow below shows a generic plugin and component for cycling through elements like a carousel.</p>
   <div class="bs-example">
-    <div id="carousel-example-generic" class="carousel slide" data-ride="carousel">
+    <div id="carousel-example-generic" class="carousel slide" data-ride="carousel" data-interval="false" data-wrap="false">
```

Comment:
I'd agree that the defaults we give in our examples should be the ones that cause the least amount of problems, and then explain to devs how they can knowingly change them, with a note to make them aware of the consequences